Source/BugsnagClient.m

```diff
@@ -931,6 +933,11 @@ - (void)notifyInternal:(BugsnagEvent *_Nonnull)event
         exc = event.originalError;
     }
 
+    // handled errors should persist any information edited by the user
+    // in a section within the KSCrash report so it can be read
+    // when the error is delivered
+    [event setOverrideProperty:@"bugsnag_event" value:[event toJson]];
```

Comment:
I think I'd prefer to see this data being kept separately to `user.overrides` to avoid blurring this relationship as it's potentially confusing - how about just `user.event`?